core/src/main/java/hudson/slaves/SlaveComputer.java

```diff
@@ -140,6 +140,12 @@
      */
     private transient int numRetryAttempt;
 
+    /**
+     * Escape hatch for rejecting connections from agents with unsupported Remoting versions
```

Comment:
This is really a new possibility than a escape hatch, right? Because the default option is still the current one.

Reply:
Yeah, the default is still the current one. Would you prefer to have it renamed to something other than a escape hatch?

Reply:
As it is now, it's not an escape hatch, so it should be called something else -- something about activating the additional validation.